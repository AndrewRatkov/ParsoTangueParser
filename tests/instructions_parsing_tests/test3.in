return "245" + g() * 8;