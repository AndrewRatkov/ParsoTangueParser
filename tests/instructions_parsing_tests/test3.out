[RETURN]--->[STR,+ ]------->[STR,"245"]
                |
                +---------->[STR,* ]------->[CALL STR g]
                                |
                                +---------->[INT,8]
